checks.d/openvpn.py

```diff
@@ -0,0 +1,48 @@
+import csv
+import os
+import datetime
+from collections import defaultdict
+
+# project
+from checks import AgentCheck
+
+# 15 seconds appears to be too short of an interval
+FILE_REFRESH_SECONDS = 25
+
+class OpenVPN(AgentCheck):
+    def check(self, instance):
+        # path and vpn_name should always be present
+        filename = instance['path']
+        vpn_name = instance['vpn_name']
+        instance_tags = instance['tags']
+        self.parse_status_file(filename, instance_tags)
+
+        if not os.path.isfile(filename):
+            self.service_check('openvpn.status.is_running', AgentCheck.CRITICAL,
```

Comment:
Datadog tends to — and this is one I think we agree on — define service check names as class vars (like your `FILE_REFRESH_SECONDS`) and reuse them in the `service_check` call.